setup.cfg

```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = pyhf_benchmark
+version = 0.0.1
+description = Benchmarking of hardware acceleration of pyhf
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pyhf/pyhf-benchmark
+author = Bo Zheng
+author_email = bozheng96@gmail.com
+license = Apache
+license_file = LICENSE
+keywords = pyhf benchmark
+project_urls =
+    Documentation = https://pyhf.github.io/pyhf-benchmark/
+    Source = https://github.com/pyhf/pyhf-benchmark/
+    Tracker = https://github.com/pyhf/pyhf-benchmark/issues
+classifiers =
+    Development Status :: 4 - Beta
+    License :: OSI Approved :: Apache Software License
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Physics
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[bdist_wheel]
+universal = 1
+
+[options]
+setup_requires =
+    setuptools_scm>=1.15.0
+    setuptools_scm_git_archive>=1.0
+package_dir =
+    = src
+packages = find:
+include_package_data = True
+python_requires = >=3.6
+install_requires =
+    click>=6.0
+    psutil~=5.7
+    pynvml~=8.0
+    shortuuid~=1.0
```

Comment:
```suggestion
```

There really is no need to have the random string at the end of the directories as the timestamp is sufficient and it isn't worth a required extra dependency. I'd just remove this and the random string bit.